.github/no-response.yml

```diff
@@ -0,0 +1,10 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
+
```

Comment:
This config means the bot will look at issues with `more information required` label on it and if it's been inactive for 7 days it will close it with the comment listed here.